src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -260,6 +263,20 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       },
     },
   },
+  Loft: {
+    description: 'Create a 3D body by blending between two or more sketches',
+    icon: 'loft',
+    needsReview: true,
+    args: {
+      selection: {
+        inputType: 'selection',
+        selectionTypes: ['solid2D'],
+        multiple: true,
+        required: true,
+        skip: false,
```

Comment:
So setting it to `true` makes the preselection flow much better, but I see why I had put it to `false` now. Not sure if it's intended or not, but setting it to `true` means it goes to the review phase after the first sketch selection in the no-preselection flow, and so it breaks it

Reply:
Ahh duh that makes sense I just ran into this with my Angle constraint work. If we were to implement a configurable callback for the selection validation this could be avoided, so +1 on [this proposal](https://github.com/KittyCAD/modeling-app/pull/4605/files#r1868333355)

Reply:
Yep 100%! As discussed on Slack I'll close this thread, and create a new issue to for a new validation prop that we can apply to more than just Loft